src/pages/index.tsx

```diff
@@ -157,31 +157,35 @@ const TaxFormSection = ({ nextRoute, isDebug, isLive }) => {
   )
 }
 
-const PostponeSection = ({ nextPostponeRoute, now, isPostponeLive, isDebug }) => (
-  <>
-    <h2 className="govuk-heading-m govuk-!-margin-top-3">
-      {`Odklad daňového priznania za rok ${TAX_YEAR}`}
-    </h2>
-    <PostponeText now={now} />
-    <ul className="govuk-list govuk-list--bullet">
-      <li>{`do 30.6.${TAX_YEAR + 1} ak ste mali príjmy len zo Slovenska, alebo`}</li>
-      <li>{`do 30.9.${TAX_YEAR + 1} ak ste mali príjmy aj zo zahraničia`}</li>
-    </ul>
-
-    {
-      (isPostponeLive || isDebug) && (
-        <>
-          <p className="govuk-body-xs">
-            Používaním tejto služby súhlasíte so spracovaním osobných údajov v rozsahu
-            nevyhnutnom na vygenerovanie odkladu daňového priznania. Vaše údaje
-            neukladáme, sú použité výlučne na spracovanie odkladu daňového priznania.
-          </p>
-          <PostponeButton now={now} nextPostponeRoute={nextPostponeRoute} />
-        </>
-      )
-    }
-  </>
-)
+const PostponeSection = ({ nextPostponeRoute, now, isPostponeLive, isDebug }) => {
+  const isPostponeExtensionTime = now.getMonth() < 11;
```

Comment:
odklad je mozne spravit iba do konca marca.

asi by som to upravil tak ze by som z `PostponeButton` vybral `isPostponeTime` do komponentu `PostponeSection`

potom mozes pouzit v `PostponeSection`  `isPostponeTime` a do `PostponeButton` uz posielat `isPostponeTime` namiesto `now`

takto ta logika bude iba na jednom mieste a iba vysledok sa bude posielat dalej dole

Reply:
Updatenute.

Tiez som si vsimol ze ten text ohladom spracovania osobnych udajov je tam dva krat, tak som to dal do novehom componentu.

Reply:
len pozor tam ten text nie je rovnaky

Reply:
ok, tak som revertol ten commit s textom o osobnych udajoch